Translated/Dialog/Sharkey/ssn_005133.csv

```diff
@@ -0,0 +1,5 @@
+name_7663#0,"""Sharkey"""
+ssn_005133_0010#0,"""...How is it going? From the looks of you, you've had some good results."""
```

Comment:
"...How was it? From the looks of you, it seems to have left an impression."

Reply:
This will need to be changed in the dialogue for Sharkey's live tasks as well.